news/2020-06-26-mappers-report-May-June.md

```diff
@@ -0,0 +1,193 @@
+
+The Beatmap Nominators never rest, the Beatmap Nominators never sleep. Just kidding, of course they do, they are still humans. However, the group size is always changing! New are coming, old are going. This month was no exception!
+
+### BN Additions
+
+- **osu!:** [Teky](https://osu.ppy.sh/users/10520912), [DeviousPanda](https://osu.ppy.sh/users/4966334), [Stixy](https://osu.ppy.sh/users/9000308), [Trynna](https://osu.ppy.sh/users/2652951), [Fall](https://osu.ppy.sh/users/4800816), [Elayue](https://osu.ppy.sh/users/6400861) & [NexusQI](https://osu.ppy.sh/users/13822800)
+- **osu!taiko:** [realy0\_](https://osu.ppy.sh/users/8863359), [\_DUSK\_](https://osu.ppy.sh/users/6092181), [davidminh0111](https://osu.ppy.sh/users/9623142), [Cychloryn](https://osu.ppy.sh/users/6921736) & [Axer](https://osu.ppy.sh/users/7299864)
+- **osu!catch:** [Baron](https://osu.ppy.sh/users/10286499), [Secre](https://osu.ppy.sh/users/2306637) & [Bastian](https://osu.ppy.sh/users/6345176)
+- **osu!mania:** [Kyousukee](https://osu.ppy.sh/users/8842107), [Shima Rin](https://osu.ppy.sh/users/6089608), [Garalulu](https://osu.ppy.sh/users/757783) & [Muse Dash](https://osu.ppy.sh/users/13695676)
+
+### BN Resignations
+
+- **osu!:** [Affirmation](https://osu.ppy.sh/users/6186628) & [wafer](https://osu.ppy.sh/users/9416836)
+- **osu!taiko:** [Cynplytholowazy](https://osu.ppy.sh/users/3901754) & [Gamelan4](https://osu.ppy.sh/users/9856910)
+- **osu!catch:** [-Luminate](https://osu.ppy.sh/users/4778689)
+- **osu!mania:** [Drum-Hitnormal](https://osu.ppy.sh/users/748722), [\_Kobii](https://osu.ppy.sh/users/6209713) & [Unpredictable](https://osu.ppy.sh/users/7560872)
+
+### BN and QAT/NAT Badges
+
+Recently a new wave of badges acknowledging the length of time members have put into teams for managing ranked! Those who are on this list have been contributing to the community in so many ways during their time on osu!, and fully deserve this acknowledgement. We thank them all for their contribution so far and hope to see much more of them in the future! Here is the list of those:
+
+- **BN 1 Year Badge:** [incandescence](osu.ppy.sh/users/6256027), [Kaitjuh](osu.ppy.sh/users/2225327), [Unpredictable](osu.ppy.sh/users/7560872), [Nuvolina](osu.ppy.sh/users/10974170), [Dored](osu.ppy.sh/users/10284894)
+- **BN 2 Year Badge:** [JBHyperion](osu.ppy.sh/users/4879508), [Nevo](osu.ppy.sh/users/7451883), [Mordred](osu.ppy.sh/users/7265097), [bossandy](osu.ppy.sh/users/360437), [Namki](osu.ppy.sh/users/5248582)
+- **BN 3 Year Badge:** [Benita](osu.ppy.sh/users/4023183), [Sonnyc](osu.ppy.sh/users/11771)
+- **BN 4 Year Badge:**  [Nozhomi](osu.ppy.sh/users/2716981)
+- **NAT 1 Year Badges:** [Ascendance](osu.ppy.sh/users/2931883), [Noffy](osu.ppy.sh/users/1541323), [Faputa](osu.ppy.sh/users/845733)
+- **NAT 2 Year Badge:** [Lasse](osu.ppy.sh/users/896613), [-Mo-](osu.ppy.sh/users/2202163), [Chaoslitz](osu.ppy.sh/users/3621552)
+- **NAT 3 Year Badge:** [OnosakiHito](osu.ppy.sh/users/290128)
```

Comment:
```suggestion
- **NAT 1 Year Badges:** [Ascendance](osu.ppy.sh/users/2931883), [Noffy](osu.ppy.sh/users/1541323), [Faputa](osu.ppy.sh/users/845733)
- **QAT/NAT 2 Year Badge:** [Lasse](osu.ppy.sh/users/896613), [-Mo-](osu.ppy.sh/users/2202163), [Chaoslitz](osu.ppy.sh/users/3621552)
- **QAT 3 Year Badge:** [OnosakiHito](osu.ppy.sh/users/290128)
```

these were specified in the list I gave before to match up with their badge descriptions, for instance ono got the badge late but was never NAT